kickoff/config.py

```diff
@@ -6,8 +6,8 @@
 LATEST_FIREFOX_OLDER_VERSION = "3.6.28"
 CURRENT_ESR = "45"
 ESR_NEXT = ""
-JSON_FORMAT_VERSION = 1
-JSON_FORMAT_L10N_VERSION = 1
+JSON_FORMAT_VERSION = 1.0
```

Comment:
that's a version number and for other version numbers in the config file, we stored them as string. if we don't intend to make calculations with these numbers, maybe we should store them as string, that would also avoid casting them to strings before using them in the template. A nitpick for consistency :)